kevinzs/day02/tests.py

```diff
@@ -0,0 +1,23 @@
+import day02
+import unittest
+
+def getInput(file):
+    input = open(file)
+    lines = input.readlines()
+    for i in range(len(lines)):
+        lines[i] = list(lines[i].strip().split())
+    input.close()
+    return lines
```

Comment:
It is nicer to have an auxiliary function for reading files than your tests of day 1. In those you had file treatment inside the tests.

Nice 👍